2024-02-19 20:29:52,032 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 20:29:52,032 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Configure stats pid to 15208
2024-02-19 20:29:52,033 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 20:29:52,033 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 20:29:52,033 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 20:29:52,033 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 20:29:52,034 INFO    MainThread:15208 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_202952-sd10izkz\logs\debug.log
2024-02-19 20:29:52,034 INFO    MainThread:15208 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_202952-sd10izkz\logs\debug-internal.log
2024-02-19 20:29:52,034 INFO    MainThread:15208 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000027159C34CA0>
2024-02-19 20:29:52,034 INFO    MainThread:15208 [wandb_init.py:init():566] calling init triggers
2024-02-19 20:29:52,035 INFO    MainThread:15208 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 10, 'max_depth': 5}
2024-02-19 20:29:52,035 INFO    MainThread:15208 [wandb_init.py:init():591] re-initializing run, found existing run on stack: fb4gzpv1
2024-02-19 20:29:52,037 INFO    MainThread:15208 [wandb_run.py:_finish():1970] finishing run mc0c0z/DecisionTree/fb4gzpv1
2024-02-19 20:29:52,038 INFO    MainThread:15208 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 20:29:52,038 INFO    MainThread:15208 [wandb_run.py:_restore():2193] restore
2024-02-19 20:29:52,038 INFO    MainThread:15208 [wandb_run.py:_restore():2199] restore done
2024-02-19 20:29:55,900 INFO    MainThread:15208 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 20:29:55,900 INFO    MainThread:15208 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 20:29:55,906 INFO    MainThread:15208 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-19 20:29:56,001 INFO    MainThread:15208 [wandb_init.py:init():616] starting backend
2024-02-19 20:29:56,001 INFO    MainThread:15208 [wandb_init.py:init():620] setting up manager
2024-02-19 20:29:56,004 INFO    MainThread:15208 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-19 20:29:56,005 INFO    MainThread:15208 [wandb_init.py:init():628] backend started and connected
2024-02-19 20:29:56,019 INFO    MainThread:15208 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-19 20:29:56,019 INFO    MainThread:15208 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-19 20:29:56,020 INFO    MainThread:15208 [wandb_init.py:init():720] updated telemetry
2024-02-19 20:29:56,021 INFO    MainThread:15208 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 20:29:57,500 INFO    MainThread:15208 [wandb_run.py:_on_init():2262] communicating current version
2024-02-19 20:29:57,922 INFO    MainThread:15208 [wandb_run.py:_on_init():2271] got version response 
2024-02-19 20:29:57,922 INFO    MainThread:15208 [wandb_init.py:init():804] starting run threads in backend
2024-02-19 20:29:58,011 INFO    MainThread:15208 [wandb_run.py:_console_start():2241] atexit reg
2024-02-19 20:29:58,011 INFO    MainThread:15208 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-19 20:29:58,011 INFO    MainThread:15208 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-19 20:29:58,012 INFO    MainThread:15208 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-19 20:29:58,012 INFO    MainThread:15208 [wandb_init.py:init():847] run started, returning control to user process
2024-02-19 20:30:20,395 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 20:30:20,396 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Configure stats pid to 15208
2024-02-19 20:30:20,396 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 20:30:20,396 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 20:30:20,396 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 20:30:20,397 INFO    MainThread:15208 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 20:30:20,465 INFO    MainThread:15208 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_203020-awod8qcc\logs\debug.log
2024-02-19 20:30:20,466 INFO    MainThread:15208 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_203020-awod8qcc\logs\debug-internal.log
2024-02-19 20:30:20,466 INFO    MainThread:15208 [wandb_init.py:init():566] calling init triggers
2024-02-19 20:30:20,467 INFO    MainThread:15208 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 2, 'max_depth': 10}
2024-02-19 20:30:20,467 INFO    MainThread:15208 [wandb_init.py:init():591] re-initializing run, found existing run on stack: sd10izkz
2024-02-19 20:30:20,469 INFO    MainThread:15208 [wandb_run.py:_finish():1970] finishing run mc0c0z/DecisionTree/sd10izkz
2024-02-19 20:30:20,469 INFO    MainThread:15208 [jupyter.py:save_history():445] not saving jupyter history
2024-02-19 20:30:20,469 INFO    MainThread:15208 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-19 20:30:20,469 INFO    MainThread:15208 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-19 20:30:20,469 INFO    MainThread:15208 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 20:30:20,470 INFO    MainThread:15208 [wandb_run.py:_restore():2193] restore
2024-02-19 20:30:20,470 INFO    MainThread:15208 [wandb_run.py:_restore():2199] restore done
2024-02-19 20:30:24,396 INFO    MainThread:15208 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 20:30:24,396 INFO    MainThread:15208 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 20:30:24,402 INFO    MainThread:15208 [wandb_run.py:_footer_sync_info():3825] logging synced files
